### PR TITLE
config working session

### PR DESCRIPTION
**Pull request by NghiaDinh003**
#What
The session can last up to 6 hours, which you can customize in the Config.toml file.

#Why
For a better user experience.

#How
image
- I just added session settings in the Config.toml file.

#Demo
- I try setting the session to 30 seconds and wait. 
- I try to ask, do not know how to hang the machine to make it automatically close the session.
- I try asking the chatbot if there is a way to test the allowed time of the session. but it instructs no way but to wait.
image

### CODE REVIEW

File: .idea/workspace.xml
Comment (namho102) [inline .idea/workspace.xml:1]: you shouldn't push this file. Learn about gitignore will help: https://git-scm.com/docs/gitignore

**Comment by namho102** [comment]
the PR title `long working session` is not easy to understand, try to write short but meaningful. Something like `config working session`  may work. Please read more about it on https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/

File: .idea/workspace.xml
Comment (NghiaDinh003) [reply .idea/workspace.xml:1]: I'm really sorry, I see showing the exclamation point. I tried to figure out what it was and mistakenly clicked merge ...
image

**Comment by NghiaDinh003** [comment]
> the PR title `long working session` is not easy to understand, try to write short but meaningful. Something like `config working session` may work. Please read more about it on https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/

**Pull request closed without merging**